lib/assessment.js

```diff
@@ -4,15 +4,30 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
+var min = function min([1, 3, 4, 2]) {
+
+  var findMin = 0;
+
+  for(var i = 0; i < min.length; i++) {
+    return Math.min.apply();
+  };
 
 };
 
+// I am not sure how to write this correctly, but I would use Math.min to find the minimum number in an array.
+
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
+var sum = function sum([1, 2, 3, 4]) {
+
+  var sumTotal = 0;
+  for(var i = 0; i < sum.length; i++); {
+    sumTotal += sum[i];
+  }
+
+  return sumTotal;
```

Comment:
When you write a function that takes an unknown number of arguments, you want to use the `arguments` array-like object which is built into each function. This object allows you to access each argument using an index value, just like an array.

In this case, you would write the following.
```javascript
var sum = function sum() {
  var sumTotal = 0;
  for (var i = 0; i < arguments.length; i++) {
    sumTotal += arguments[i];
  }
  return sumTotal;
};
```

Otherwise, your logic for approaching this problem is spot on.